### ISSUE TITLE
"vistibility" is always "unlisted"

### ISSUE THREAD

**Issue by selyod-ka** [robertoszek/pleroma-bot#86]
Hey, thanks for your work on pleroma-bot!

I noticed that all toots get the visibility "unlisted" despite stated otherwise in my config.yml.
In my understanding, the reason is that `default_cfg_attributes` [misses the key "visibility"](https://github.com/robertoszek/pleroma-bot/blob/4749ce3eff091d2bdd89354763869ed54a2bae55/pleroma_bot/cli.py#L100-L129) and therefore [`if not hasattr(self, "visibility"):`](https://github.com/robertoszek/pleroma-bot/blob/4749ce3eff091d2bdd89354763869ed54a2bae55/pleroma_bot/cli.py#L211) is always true, i.e. "visibility" is set to "unlisted".

**Comment by robertoszek** [comment]
Hi!
What version are using? `1.0.2`?

Would you mind trying out `1.0.3rc14`? There was a [regression fixed about visibility](https://github.com/robertoszek/pleroma-bot/blob/2ee52c758bd28d0da1b0172c68192689a6164a31/CHANGELOG.md?plain=1#L4) when defined as a global mapping in the config, which I'm not sure if it's your case.

You can install the release candidate like so:
```
pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pleroma-bot==1.0.3rc14
```

If still doesn't work as expected, report back and please provide your config (excluding any sensitive data, like tokens and so on) and I'll try to reproduce it on my end.

**Comment by selyod-ka** [comment]
Hey!

Yes, I'm on `1.0.2` and I'm using 'visibility' in the global mapping.

Have not tried `1.0.3rc14`, but downloaded it locally and checked if 'visibility' is a key of 'default_cfg_attributes'. As that is the case, I'm sure this issue is fixed in `1.0.3rc14`.
Thanks.

**Comment by robertoszek** [comment]
Even so, if you find it doesn't work as expected on `1.0.3rc14` (or 1.0.3 stable, whenever it releases), feel free to open a new issue and I'll check it out.
Have a nice day!
